src/pages/Categories.js

```diff
@@ -1,10 +1,9 @@
-import React from 'react';
-
 const Categories = () => (
-  <div>
-    <h1>Categories</h1>
-    <button type="button">Check Status</button>
-  </div>
+  <>
+    <div className="alert alert-warning">
+      <h2>Under construction</h2>
+    </div>
+  </>
```

Comment:
- [ ] it would be better if there is a button called `check status ` that will dispatch an action to display under construction  but on your page you just displayed plain text so please try to fix that in order to match the requirements 
<img width="1440" alt="Screenshot 2023-01-02 at 15 14 17" src="https://user-images.githubusercontent.com/61790587/210243277-7b7b6bcc-2718-45ec-8984-2a5b7084ecb6.png">